src/schemas/settings/v4/index.ts

```diff
@@ -0,0 +1,129 @@
+import { z } from 'zod'
+
+const RegularProxySettingsSchema = z.object({
```

Comment:
Previous migrations were all referencing a bunch of schemas in `v1`. This makes it easy to accidentally change previous versions of the schema. From `v4` and onward each version should contain a complete copy of the entire schema.